easybuild/tools/toolchain/toolchain.py

```diff
@@ -584,6 +586,24 @@ def _verify_toolchain(self):
             raise EasyBuildError("List of toolchain dependency modules and toolchain definition do not match "
                                  "(found %s vs expected %s)", self.toolchain_dep_mods, toolchain_definition)
 
+    def symlink_compilers(self, paths):
+        """
+        Create a symlink for each compiler to binary/script at specified path.
+
+        @param paths: dictionary containing mapping from types of caches (ccache, f90cache) to
+                      tuple ('path/to/cache', [commands to symlink to this cache])
+        """
+        tmpdir = tempfile.mkdtemp()
+
+        for key, links in paths.items():
```

Comment:
I would do `for _, (path, comps) in paths.items()` to make this more readable?

also, key is not used?!